src/node/modules.h

```diff
@@ -10,18 +12,6 @@
 
 namespace ccf
 {
-  struct Module
-  {
-    std::string js;
-
-    Module() = default;
-
-    Module(const std::string& js_) : js(js_) {}
-
-    MSGPACK_DEFINE(js);
-  };
-  DECLARE_JSON_TYPE(Module)
-  DECLARE_JSON_REQUIRED_FIELDS(Module, js)
-
-  using Modules = kv::Map<std::string, Module>;
+  using Module = std::string;
+  using Modules = kv::RawCopySerialisedMap<std::string, Module>;
```

Comment:
FYI, using `ServiceMap` has some non-negligible impact (~-15%) on the `ls_js_sgx_cft` perf test.

Reply:
This table is a prime candidate to benefit from #2220. Large strings to deserialise, changed rarely, used by every endpoint invocation.

Reply:
There's also a considerable amount of unnecessary duplication because of the export hack at the moment.